apps/wallet-mobile/src/TxHistory/TxHistoryNavigator.tsx

```diff
@@ -190,11 +191,42 @@ export const TxHistoryNavigator = () => {
 
                     <Stack.Screen
                       name="exchange-create-order"
-                      component={CreateExchangeOrder}
                       options={{
-                        title: strings.rampOnOffTitle,
+                        title: strings.exchangeCreateOrderTitle,
                       }}
-                    />
+                    >
+                      {() => (
+                        <Boundary>
```

Comment:
we will be using suspense or the general callbacks? I'm asking cuz this will cause the  entire screen to go with the spinner only, basically <View><Spinner /></View>

Reply:
Waiting Product feedback